nnbuilder/src/test/kotlin/org/hse/nnbuilder/queue/TaskExecutorTest.java

```diff
@@ -0,0 +1,27 @@
+package org.hse.nnbuilder.queue;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SpringBootTest
+@ActiveProfiles("test")
+public class TaskExecutorTest {
+
+    @Autowired
+    TaskExecutor taskExecutor;
+
+    @Test
+    void callProcessTaskTest() {
+        taskExecutor.processTask();
```

Comment:
So what does it test? :)

It should be reasonable to test the logic with mocks.